src/forms/CreateEmployee/CreateEmployeeForm.jsx

```diff
@@ -128,9 +128,10 @@ export const CreateEmployeeForm = ({
         workingShift: data.timeShift,
         role: data.jobPosition //TODO: see what the BE needs here
       });
-  
-      if (response.success) {
+
+      if (response && response.message === 'User created successfully!') {
```

Comment:
lo tuve q hacer asi pq la response solo tiene el "message".
probé `response.success` o `response.status === 200` pero no anda. no es lo mejor pero como anda no me preocuparía la verdad de q el BE lo arregle.

Reply:
por como esta implemenato el commonFetch no vas a poder acceder a esa info. Antes si podiamos porque el backend la enviaba a nivel body. Lo que vamos a hacer mas adelante es que cuadno no nos devuelva el status que necesitemos (generalmente 2xx) vamos a lanzar una excepcion dsde el commonFetch y la vamos a capturar en el useEffect para mostrar la xp de error. Por ahora, saquemos la validacion y asumamos que responde bien

Reply:
En general, excelente Santi 👍